modules/ROOT/pages/runtime-patching.adoc

```diff
@@ -30,19 +29,20 @@ By default, MUnit runs with the runtime corresponding to the _minMuleVersion_ of
     </configuration>
 </plugin>
 ----
-<1> Remember that this version is an example. You need to declare the correct version provided by the team.
+<1> This version is a placeholder example. You must declare the correct version provided by MuleSoft.
 
-== Patching MUnit from Studio
+== Patching MUnit Using Studio
 
-By default, MUnit runs in studio with the version of the project’s server. You can override this version, by specifying the patched runtime version in the MUnit run configuration:
+By default, MUnit runs in Anypoint Studio with the Mule runtime engine version of the project’s server. You can override this version by specifying the patched Mule version in the MUnit run configuration menu:
 
 . In the top menu bar, go to *Run* and *Run Configurations…*.
-. Under MUnit, select your MUnit test file.
-. Type the MUnit patched version you want to use in the *Mule Runtime Version* field, under *Advanced Settings*.
+. Under *MUnit*, select your MUnit test file.
+. In the *Mule Runtime Version* field under *Advanced Settings* type the MUnit patched version you want to use.
 +
 image::runtime-patching-7d385.png[]
-+
- If you already set the runtimeVersion attribute in the pom as shown in the “From Maven” section, you can click the *Load configuration from pom file* option to automatically load this version into the *Mule Runtime Version* field.
+
+
+If you already set the `runtimeVersion` attribute in the POM file of your project as shown in previous Maven instructions, you can click the *Load configuration from POM file* option to automatically load this version into the *Mule Runtime Version* field.
```

Comment:
```suggestion
If you already set the `runtimeVersion` attribute in the POM file of your project, click *Load configuration from POM file* to automatically load this version into the *Mule Runtime Version* field.
```

Reply:
An alternative presentation for the final step is...:

3. If you have already set `runtimeVerions` in your project POM file, click...to automatically load...
    Otherwise, type the number of the version that you want to use in that field.